db/pgstorage/migrations/0001.sql

```diff
@@ -1,114 +1,110 @@
 -- +migrate Down
-DROP SCHEMA IF EXISTS syncv2 CASCADE;
-DROP SCHEMA IF EXISTS mtv2 CASCADE;
+DROP SCHEMA IF EXISTS sync CASCADE;
+
+DROP SCHEMA IF EXISTS mt CASCADE;
```

Comment:
you have to be careful because we are already in public. Maybe table and schema modifications should be in a different migration to allow upgrade from the previous version

Reply:
sure, I will care next time, but curious if there is a way to migrate the existing database, because we already modified the rht table schema, we need to re-sync from the scratch, right?

Reply:
yes, If we modify the schema or the tables in the initial migration, we will need to resync